server/views/sources/collectionedit.py

```diff
@@ -324,7 +324,7 @@ def update_metadata_for_sources(source_list):
         TagSetDiscoverer().media_subject_country_set: 'subject_country',
         TagSetDiscoverer().media_type_set: 'media_type',
     }
-    for tags_id in TagSetDiscoverer().media_metadata_sets:
+    for tags_id in TagSetDiscoverer().media_metadata_sets():
```

Comment:
yeah @kilemensi  it's being used in a couple more places. Also, since we will want to fetch upstream at some point, I think we can try remaining as close as MC code as possible.

Reply:
If it's used in multiple places, then yes, lets follow whatever call convention is used in those other places @esirK